### PR TITLE
Fix some issues that were found in testing environment

### PR DESCRIPTION
**Pull request by Ngriffiths1**
Added some logging for potential errors, changed html to only display relevant fields and give more of an explanation when requested functional scopes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ This is an example application of a third party app gaining an Oauth token and u
 * Entrypoint is http://localhost:8090/login
 * Once every 5 minutes the application attempts a User not present journey with all the users who have logged in, and uses the refresh token if required.
 * For CHS usage, environment variables have already been created internally to run app locally or in team environments
+* Whilst using this app it is only configured to work with HTTP
```
Comment (rafahop) [inline README.md]: I think this depends on how the server where the app is running is configured so probably not something to be in the readme

File: src/main/resources/templates/loginRequestedScope.html [hunk 2]
```diff
@@ -13,7 +13,9 @@
     <div class="jumbotron jumbotron-fluid">
       <div class="container">
         <h1 class="display-6">Proof of concept application</h1>
-        <p class="lead">Enter the requested scopes you wish to obtain.</p>
+        <p class="lead">Enter the requested scopes you wish to obtain. To file with Companies House you must also obtain the user_profile : read scope.
```
Comment (rafahop) [inline src/main/resources/templates/loginRequestedScope.html]: `user_profile : read` is a permission, not a scope. Should we refer to a valid scope instead?

Would it be a good idea prepopulating the scope input field with that value?

What happens if the user does not enter that scope? Should we handle that scenario?

**Review by rafahop — CHANGES_REQUESTED** [review]
Do we really want those new debug log lines? I thought they were there for your own local debugging but if we really want some logging we should make it meaningful and definitely not log any sensitive data (and the access token is very sensitive)

File: README.md (hunk 1 above)
Comment (Ngriffiths1) [reply README.md]: Billy wanted me to put a line in the readme to say that the app in configured to work over HTTP, if you have a better suggestion for the information i can put that in instead?

File: src/main/resources/templates/loginRequestedScope.html (hunk 2 above)
Comment (Ngriffiths1) [reply src/main/resources/templates/loginRequestedScope.html]: Changed the text to be more accurate and prepopulated the text area with the relevant scope. If they do not enter that scope then they haven't read the text area correctly.

File: README.md (hunk 1 above)
Comment (Ngriffiths1) [reply README.md]: Put a more accurate message in and removed the CHS line

**Review by rafahop — APPROVED** [review]

**Comment by arslan-raza-143** [comment]
b011ae64efd88d24be0a0542abfbb934521faca3

**Pull request merged**